main.go

```diff
@@ -410,6 +423,37 @@ func run(c *cli.Context) error {
 		return fmt.Errorf("Error: %s\n", err)
 	}
 
+	rolloutCheck := c.StringSlice("rollout_check")
+	rolloutTimeout := c.Int("rollout_timeout")
+
+	log(fmt.Sprintf("Running rollout check for %v\n", rolloutCheck))
+
+	for _, deployment := range rolloutCheck {
+		command := []string{
+			"-t",
+			strconv.Itoa(rolloutTimeout),
+			kubectlCmd,
+			"rollout",
+			"status",
+			"deploy",
+			deployment,
+		}
+
+		if namespace != "" {
+			command = append(command, "--namespace", namespace)
+		}
+
+		if rolloutTimeout == 0 {
+			err = runner.Run(kubectlCmd, command[3:]...)
+		} else {
+			err = runner.Run(timeoutCmd, command...)
```

Comment:
not that big of a deal, but I find all this handling of the command a little confusing.